magento/pwa-studio
# [Fix bug]: Clicking on Label should focus Input

## Description

Add a product to the cart, head to checkout. Clicking on any label should put the focus on the associated input.

## Related Issue





Closes #2773.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Add any product to cart.
2. Navigate to checkout page.
3. Make sure when you click to label in email, first name, last name, ... on form, the cursor mouse will be focused to refer.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
